docs/knowledge-sharing/local-elasticsearch.md

```
@@ -0,0 +1,72 @@
+
+---
+
+## Prereqs
+
+- **Docker Desktop** installed and running.
+- Set Linux kernel `vm.max_map_count` **inside** Docker Desktop’s VM/WSL before starting ES:
+  make es-host-sysctl
+  ```
+-(Our Makefile does the platform-specific work for macOS/Windows. You don’t run sudo on your host.)
+
+### Build & Run
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)